src/main/javascript/crypto/src/libs/Attestation.ts

```diff
@@ -64,6 +65,13 @@ export class Attestation {
         if (decodedAttestationObj.validity){
             me.notValidBefore = decodedAttestationObj.validity.notBefore.generalizedTime.getTime();
             me.notValidAfter = decodedAttestationObj.validity.notAfter.generalizedTime.getTime();
+            // TODO validate time when it will be updated in Java code
```

Comment:
It should be updated in the java code, so I think the code fragment can be enabled.